Glideslope.json

```diff
@@ -22399,6 +22399,19 @@
    "glideslope": 3.00307574,
    "receptionRange": 10
  },
+ {
+   "name": "GS",
+   "airportICAO": "KPUW",
+   "identifier": "IPUW",
+   "associatedRunwayNumber": "5",
+   "frequency": 110350,
+   "elevation": 2566,
+   "latitude": 46.7407203,
+   "longitude": -117.12223469,
+   "bearing": 70.06,
+   "glideslope": 3.007006,
```

Comment:
This should be 3.0007006 because the heading is 070.06

Reply:
Fixed.